multic/cli/MultiCompartmentSegment/MultiCompartmentSegment.py

```diff
@@ -1,23 +1,102 @@
 def main(args):
 
     folder = args.base_dir
     base_dir_id = folder.split('/')[-2]
     _ = os.system("printf '\nUsing data from girder_client Folder: {}\n'".format(folder))
     print('new version')
-    _ = os.system("printf '\n---\n\nFOUND: [{}]\n'".format(args.input_file))
+    _ = os.system("printf '\n---\n\nFOUND: [{}]\n'".format(args.files))
 
     cwd = os.getcwd()
     print(cwd)
     os.chdir(cwd)
 
-    cmd = "python3 ../segmentationschool/segmentation_school.py --option {} --base_dir {} --modelfile {} --girderApiUrl {} --girderToken {} --files {}".format('predict', args.base_dir, args.modelfile, args.girderApiUrl, args.girderToken, args.input_file)
-    print(cmd)
-    sys.stdout.flush()
-    os.system(cmd)
+    #cmd = "python3 ../segmentationschool/segmentation_school.py --option {} --base_dir {} --modelfile {} --girderApiUrl {} --girderToken {} --files {}".format('predict', args.base_dir, args.modelfile, args.girderApiUrl, args.girderToken, args.input_file)
+    #print(cmd)
+    #sys.stdout.flush()
+    #os.system(cmd)
+    default_vals = {
+        'girderApiUrl':' ',
+        'girderToken':' ',
+        'files':' ',
+        'option':'predict',
+        'transfer':' ',
+        'one_network':True,
+        'target':None,
+        'cortextarget':None,
+        'output':None,
+        'wsis':None,
+        'groupBy':None,
+        'patientData':None,
+        'labelColumns':None,
+        'labelModality':None,
+        'IDColumn':'Label_slides',
+        'plotFill':True,
+        'scatterFreatures':'5,6',
+        'anchor':'Age',
+        'exceloutfile':None,
+        'SummaryOption':None,
+        'base_dir': os.getcwd(),
+        'code_dir':os.getcwd(),
+        'gpu_num':2,
+        'gpu':0,
+        'iteration':'none',
+        'prune_HR':0.0,
+        'prune_LR':0.0,
+        'classNum':0,
+        'classNum_HR':0,
+        'modelfile':None,
+        'white_percent':0.01,
+        'chop_thumbnail_resolution':16,
+        'overlap_percentLR':0.5,
+        'boxSizeLR':450,
+        'downsampleRateLR':16,
+        'overlap_percentHR':0,
+        'boxSize':2048,
+        'downsampleRateHR':1,
+        'training_max_size':512,
+        'Mag20X':False,
+        'aug_HR':3,
+        'aug_LR':15,
+        'hbound':0.01,
+        'lbound':0.025,
+        'CNNbatch_sizeLR':2,
+        'CNNbatch_sizeHR':2,
+        'epoch_LR':1,
+        'epoch_HR':1,
+        'saveIntervals':10,
+        'learning_rate_HR':2.5e-4,
+        'learning_rate_LR':2.53-4,
+        'chop_data':'false',
+        'crop_detectron_trainset':False,
+        'predict_data':True,
+        'roi_thresh':0.01,
+        'outDir':'Predictions',
+        'save_outputs':False,
+        'imBoxExt':'.jpeg',
+        'finalImgExt':'.jpeg',
+        'wsi_ext':'.svs,.scn,.ndpi',
+        'bg_intensity':0.5,
+        'approx_downsample':1,
+        'min_size':[30,30,30,30,30,30],
+        'bordercrop':300,
+        'LR_region_pad':50,
+        'show_interstitium':True
+    }
```

Comment:
We can move this outside of the function to the top. You can also call it `DEFAULT_VALS`. As default `args` are now stored in this dictionary, we don't need to parse them in `segmentation_school` right? If so let's delete parsing in `segmentation_school`.